datadog_operator/metadata.csv

```diff
@@ -1,2 +1,69 @@
+datadog.operator.controller_runtime_reconcile_time_seconds.sum,count,,second,,Sum of time per reconciliation per controller,0,datadog_operator,cr_reconcile_time_seconds_s,
+datadog.operator.controller_runtime_reconcile_total,count,,,,Total number of reconciliations per controller,0,datadog_operator,cr_reconcile,
+datadog.operator.default.feature.enabled,gauge,,,,,0,datadog_operator,default_enabled,
+datadog.operator.dogstatsd.feature.enabled,gauge,,,,`1` if the DogStatsD feature is enabled,0,datadog_operator,dogstatsd_enabled,
+datadog.operator.eds_controller_leader,gauge,,,,,0,datadog_operator,eds_controller_leader,
+datadog.operator.event_collection.feature.enabled,gauge,,,,`1` if the Event Collection feature is enabled,0,datadog_operator,event_collection_enabled,
+datadog.operator.go_gc_duration_seconds.count,count,,second,,Count of the GC invocation durations,0,datadog_operator,go_threads_c,
+datadog.operator.go_gc_duration_seconds.quantile,gauge,,second,,Quantiles of the GC invocation durations,0,datadog_operator,go_threads,
+datadog.operator.go_gc_duration_seconds.sum,count,,second,,Sum of the GC invocation durations,0,datadog_operator,go_threads_s,
+datadog.operator.go_goroutines,gauge,,thread,,Number of goroutines that currently exist,0,datadog_operator,go_goroutines,
 datadog.operator.go_info,gauge,,,,Go version,0,datadog_operator,go_info,
+datadog.operator.go_memstats_alloc_bytes,gauge,,byte,,Number of bytes allocated and still in use,0,datadog_operator,go_memstats_alloc_bytes,
+datadog.operator.go_memstats_alloc_bytes_total,count,,byte,,Total number of bytes allocated even if freed,0,datadog_operator,go_memstats_alloc_bytes_total,
+datadog.operator.go_memstats_buck_hash_sys_bytes,gauge,,byte,,Number of bytes used by the profiling bucket hash table,0,datadog_operator,go_memstats_buck_hash_sys_bytes,
+datadog.operator.go_memstats_frees_total,count,,,,Total number of frees,0,datadog_operator,go_memstats_frees_total,
+datadog.operator.go_memstats_gc_cpu_fraction,gauge,,,,The fraction of this program's available CPU time used by the GC since the program started,0,datadog_operator,go_memstats_gc_cpu_fraction,
+datadog.operator.go_memstats_gc_sys_bytes,gauge,,byte,,Number of bytes used for garbage collection system metadata,0,datadog_operator,go_memstats_gc_sys_bytes,
+datadog.operator.go_memstats_heap_alloc_bytes,gauge,,byte,,Number of heap bytes allocated and still in use,0,datadog_operator,go_memstats_heap_alloc_bytes,
+datadog.operator.go_memstats_heap_idle_bytes,gauge,,byte,,Number of heap bytes waiting to be used,0,datadog_operator,go_memstats_heap_idle_bytes,
+datadog.operator.go_memstats_heap_inuse_bytes,gauge,,byte,,Number of heap bytes that are in use,0,datadog_operator,go_memstats_heap_inuse_bytes,
+datadog.operator.go_memstats_heap_objects,gauge,,,,Number of allocated objects,0,datadog_operator,go_memstats_heap_objects,
```

Comment:
you could use `object` unit

Reply:
Addressed by https://github.com/DataDog/integrations-core/pull/15655/commits/facfe66276ae175e6b0fb0244f78b2fb21d0677f